broker/applications/osb-broker/src/server.js

```diff
@@ -28,10 +28,6 @@ async function init() {
         });
       });
       app.use('/:platform(cf|k8s|sm)', routes.broker);
-      if (!_.includes(config.disabled_apis, 'admin')) {
```

Comment:
Please also remove the occurrence of disabled_apis from every place, I guess it is part of the helm settings as well.